src/pages/verify/integrations/react-native-expo.mdx

```diff
@@ -0,0 +1,113 @@
+---
+product: verify
+category: Integrations
+sort: 0
+title: React Native | Expo
+subtitle: Accept MitID, NemID, Swedish BankID, Norwegian BankID and more eID logins in your Expo (React Native) app with @criipto/verify-expo.
+---
+
+import Layout from '../../../layouts/mdx';
+export default Layout;
+
+Read the [full documentation](https://github.com/criipto/criipto-verify-expo) or [download a sample application from GitHub](https://github.com/criipto/criipto-verify-expo/tree/master/example).
+
+## App switch support
```

Comment:
This section should be rewritten to be more focused on being an implementation guide I think.
The `supports` statement doesn't matter in a guide I think, but we can move the `Danish MitID + Android` to later in the guide, after installation/implementation.